### DISCUSSION TITLE
Which UI framework is used?

### DISCUSSION THREAD

**Discussion by goyalyashpal** [stefankueng/BowPad#369] [Q&A] [answered]
hi!

i was just wondering which UI framework is used in here? it's none of C# .Net,  Lazarus Pascal, electron, GTK, Qt.

can it be cross platform? is it wxWidgets? 

i ain't very familiar with C++ projects, so, couldn't see right away. but i tried looking in several files like: [`src/MainWindow.h`][main], `src/UICollection.h`, but erhh, couldn't figure out much else.

[main]: https://github.com/stefankueng/BowPad/blob/main/src/MainWindow.h

----
P.S.: 
the [DemoHelper](https://tools.stefankueng.com/DemoHelper.html) is looking interesting too (: 

oh, and i came here as:
* <- "Visit Stefan Kung's website"
* <- About
* <- `grepWinNP3` autocomplete suggestion in powershell.

**Comment by stefankueng** [comment, marked as answer]
no ui framework. Just plain windows APIs.

**Reply by goyalyashpal** [reply]
😱 awesome
